src/pages/ProfileSettings.tsx

```diff
@@ -1,20 +1,261 @@
+                          )}
+                          <Label htmlFor="profilePic" className="relative">
+                            {image ? (
+                              <img src={image} className="h-15 w-15 rounded-full p-1 z-0" />
+                            ) : (
+                              <UserRound className="h-15 w-15 rounded-full bg-blue text-white lg:p-3 p-1" />
+                            )}
+                            <Pencil className="bg-white border border-gray-300 h-6 w-6 p-1 z-10 rounded-full absolute bottom-1 right-1 text-gray-400" />
+                            <Input
+                              type="file"
+                              id="profilePic"
+                              placeholder=""
+                              onChange={onImageChange}
+                              className="hidden"
+                            />
+                          </Label>
+                        </div>
+
+                        <div className="flex-1 gap-4 justify-center text-left mt-8 flex-col lg:gap-4 w-full">
+                          <div className="flex flex-col lg:flex-row lg:gap-x-4">
+                            <div className="lg:w-1/2">
+                              <InputWithLabel
+                                label="First Name"
+                                type="text"
+                                placeholder="Type here ..."
+                                classNameDiv="mt-2"
+                              />
+                            </div>
+                            <div className="lg:w-1/2">
+                              <InputWithLabel
+                                label="Last Name"
+                                type="text"
+                                placeholder="Type here ..."
+                                classNameDiv="mt-2"
+                              />
+                            </div>
+                          </div>
+                          <InputWithLabel
+                            label="Company Name"
+                            type="text"
+                            placeholder="Type here ..."
+                            classNameDiv="mt-2"
+                          />
+                          <InputWithLabel
+                            label="Email"
+                            type="email"
+                            placeholder="emailid@gmail.com"
+                            classNameDiv="mt-2"
+                          />
+                          <InputWithTagLeft
+                            title="Mobile Number"
+                            tag="+91"
+                            placeholder="Type here ..."
+                            classNameDiv="mt-2"
+                          />
+                        </div>
+                      </div>
+                      <AlertDialogFooter className="px-6 py-2 flex flex-row items-center border-t gap-4 justify-end">
+                        <AlertDialogCancel
```

Comment:
Just Use two simple buttons for Save and Cancel actions instead of alertDialogButtons